README.md

```diff
@@ -55,7 +55,7 @@ By participating, you are expected to uphold this code. Please [report](https://
 unacceptable behavior as soon as possible.
 
 ## License
-
+ [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)  
```

Comment:
Hi @EricBrianAnil, thanks for this PR. Could you please explain why you've added this line?

Reply:
Yess sure, It's the License badge. I've noticed that most official repos have the badge that redirects it to the original Apache License file. so yea :p

Reply:
Please remove this line, our license section explains that we use different licenses for different parts of the code, so not just the apache license.